### PR TITLE
CSS: deleted useless page

### LINKED ISSUE #14458: "CSS Properties Reference":  "border-radius" and its constituent properties are missing
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference

#### What information was incorrect, unhelpful, or incomplete?

CSS property "border-radius" and its constituent properties are missing from the CSS properties list.

#### Specific section or headline?

The CSS properties table.

#### What did you expect to see?

Entries (table rows) for 
[border-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius)
[border-top-left-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-top-left-radius)
    [border-top-right-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-top-right-radius)
    [border-bottom-right-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom-right-radius)
    [border-bottom-left-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom-left-radius)

#### Did you test this? If so, how?

Searched page for "border-radius" with result "phrase not found".

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/css_properties_reference`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/css_properties_reference/index.md
* Last commit: https://github.com/mdn/content/commit/26f50bdaeb1228f458d21dbdf7dc110387b20daa
* Document last modified: 2021-08-12T23:59:09.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
Page was useless. Resolves https://github.com/mdn/content/issues/14458
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 # FROM-URL	TO-URL
 /en-US/docs/-moz-locale-dir(ltr)	/en-US/docs/Web/CSS/:-moz-locale-dir(ltr)
 /en-US/docs/-moz-locale-dir(rtl)	/en-US/docs/Web/CSS/:-moz-locale-dir(rtl)
+/en-US/docs//Web/CSS/CSS_Properties_Reference	/en-US/docs//Web/CSS/Reference#index
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:9]: I think this is the reason of the test failure:
```suggestion
/en-US/docs//Web/CSS/CSS_Properties_Reference	/en-US/docs/Web/CSS/Reference#index
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (estelle) [inline files/en-us/_redirects.txt:9]:
```suggestion
/en-US/docs//Web/CSS/CSS_Properties_Reference	/en-US/docs/Web/CSS/Reference#index
```

**Comment by estelle** [comment]
unable to commit or edit the redirects. are you able to?

**Pull request closed without merging**